spec/requests/api/auth/journalist_can_create_articles_spec.rb

```diff
@@ -0,0 +1,110 @@
+RSpec.describe "POST /api/articles", type: :request do
+  let(:journalist) { create(:user, role: "journalist") }
+  let(:auth_headers) { journalist.create_new_auth_token }
+
+  describe "successfully" do
+    before do
+      post "/api/articles",
+           params: {
+             article: {
+               title: "Obnoxious Title",
+               teaser: "Some damn teaser",
+               body: "Husband found dead allegedly because he wasn't testing first",
+               category: "Hollywood",
+               user_id: journalist.id,
```

Comment:
There is no need to pass in `user_id`, you are authenticating with a token and this attribute is not used in your implementation.

Reply:
Ye, not sure what's going on there actually :3